README.md

```diff
@@ -1,2 +1,66 @@
 # AddressValidation
 AddressValidation for Magento 2
+
+Introduction
+------------
+Goal is to make an extension that can both lookup and validate customer addresses
+This extension should solve the problem that orders are shipped to non-existing addresses
+The extension will be a bridge between various address lookup services and magento
+
+Approach
+--------
+- We do the assumption that if an address can be looked up it must be a valid address
+- We do not use a special "housenumber" address attribute but make usage of the different address lines (configurable)
+
+Address lookup service interface parameters
+-------------------------------------------
+Fields needed for looking up address (multiselect)
+- Country (always needed)
+- State (optional)
+- Address (optional
+- Housenumber (optional)
+- postcode (optional)
+
+List of countries that are allowed/supported by the service (multiselect)
```

Comment:
Empty list. For now you can probably leave it out, as it has no specific importance for the first iteration of the extension.